cypress/integration/3_navigations_spec.js

```diff
@@ -1,9 +1,59 @@
+/* eslint-disable no-undef */
 /// <reference types="cypress" />
 
 describe('navigation', () => {
   beforeEach('Login', () => {
     cy.login();
   });
+  it('uses Dashboard Card to navigate to ALL engagements', () => {
+    cy.visit('/app/dashboard');
+    cy.contains('Dashboard');
+
+    cy.get('[data-cy=button_all]').should('exist');
+    cy.get('[data-cy=button_all]').click();
+
+    cy.get('[data-cy=engagement_cards_section]')
+      .find('[data-cy=active]')
+      .its('length')
+      .should('be.gte', 0);
+  });
```

Comment:
This is breaking, and seems to not be navigating to the the right page, though the others might be. In fact, it might not be working at all looking at the tests.